web_src/css/repo/issue-list.css

```diff
@@ -1,15 +1,61 @@
-.issue-list-toolbar {
-  display: flex;
-  flex-wrap: wrap;
-  align-items: flex-start;
-  gap: 1rem;
-  margin-top: 1rem;
+@media only screen and (min-width: 1024px) and (max-width: 1266px) {
+  #issue-filters > .column.six.wide.column {
+    max-width: fit-content;
+  }
+
+  #issue-filters > .issue-list-toolbar-right > .stackable.menu {
+    margin-left: calc(33% - 7vw);
+  }
+}
+
+@media only screen and (min-width: 1266px) {
+  #issue-filters > .issue-list-toolbar-right > .stackable.menu {
+    margin-left: 14vh;
+  }
 }
 
-.issue-list-toolbar-left {
-  display: flex;
+@media only screen and (max-width: 767.98px) {
+  .ui.container > .ui.stackable.grid > .column.issue-list-toolbar-right > .stackable.menu {
+    overflow-y: hidden;
+    overflow-x: scroll;
+    flex-direction: row !important;
+    flex-wrap: nowrap !important;
+  }
```

Comment:
Too long selectors, give it a unique class instead.

Reply:
I have changed to shorter and concise query selector.